### ISSUE TITLE
Replace use of 'STATIC' with 'static' (Bugzilla Bug 1767)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/edk2#7930]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 1767
Date: 2019-04-29T12:34:15+00:00
From: @mdkinney
To:   @LiuZhiguang001
CC:   @lersek, yonghong.zhu

Depends on:   #1766
Last updated: 2020-02-27T12:55:08+00:00

**Comment by tianocore-issues** [comment]
## Comment 7342
Date: 2019-04-29 12:34:15 +0000
From: @mdkinney

- Industry Specification: ---
- Release Observed: EDK II Master
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @LiuZhiguang001

Update C code in all packages to use &apos;static&apos; instead of &apos;STATIC&apos;.

This is based on the following email thread:

https://edk2.groups.io/g/devel/topic/patch_v2_3_8/31318888

This C code update also depends on an update to the EDK II C Coding Standard Specification that removes the use of the &apos;STATIC&apos; macro.

This code change is not a high priority, and should be done package at a time.

**Comment by tianocore-issues** [comment]
## Comment 7344
Date: 2019-04-29 12:44:57 +0000
From: @mdkinney

Also review all functions and global variables.  If they are not referenced
outside the current C file, then make them &apos;static&apos;.

**Comment by tianocore-issues** [comment]
## Comment 7351
Date: 2019-04-29 23:35:12 +0000
From: Yonghong Zhu &lt;<yonghong.zhu>&gt;

zhiguang will help on it.

**Comment by tianocore-issues** [comment]
## Comment 11463
Date: 2020-02-27 12:55:08 +0000
From: @lersek

The latest commit on the edk2-CCS master branch is f5ad35ec2c6d (&quot;must comment: add rule for documenting spurious variable assignments&quot;, 2019-09-18); as of that commit, the CCS still has two instances of &quot;STATIC&quot;. This means we&apos;ve had no capacity for pre-requisite bug 1766 in 8-9 months, or for this bug. Closing as WONTFIX.
